### PR TITLE
Added two new quotes

### PR DESCRIPTION
**Pull request by RaithZx**
## ✅ New Quote Checklist
- [x ] The quote(s) added are in alphabetical order and not ❌ duplicates.
- [x ] The quote(s) added are about **learning a foreign language**.
- [x ] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [x ] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

This PR references #13

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -447,6 +447,14 @@
     "author": "Safir Kassim Boudjelal",
     "sentence": "You want the 3 golden rules to learn any language? Use it or lose it. Love it or leave it. Feel it or fail it."
   },
+  {
+    "author": "Safir Kassim Boudjelal",
+    "sentence": "Good writers are good readers and listeners."
```
Comment (Kvaibhav01) [inline server/data/quotes.json:452]: This quote is not related to "learning a foreign language" or something about language. Please change it. It doesn't meet the [requirements](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_quote.md#-checklist).

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Thanks for the PR @RaithZx. Please update as suggested.

**Review by jayehernandez — CHANGES_REQUESTED** [review]
Forgot to request changes, please refer to @Kvaibhav01's comment for changes needed.

**Comment by jayehernandez** [comment]
Closing stale PR

**Pull request closed without merging**